cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -91,9 +91,13 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 
 	streamOptimizedPath := ""
 
+	// NOTE: Currently OSBuild supports multiple exports, but this isn't used
+	// by any of the image types and it can't be specified during the request.
+	// Use the first (and presumably only) export for the imagePath.
+	exportPath := args.Exports[0]
```

Comment:
Keep in mind we need to handle the case of old composer with new workers. Maybe handle this the same as in the koji case if `len(args.Exports)==0`?

Should we could also refuse jobs with `len(args.Exports)>1`? If someone is requesting two artifacts and we return only one, something is not right...

Reply:
> Keep in mind we need to handle the case of old composer with new workers. Maybe handle this the same as in the koji case if `len(args.Exports)==0`?

Good point.  I'll cover the compatibility case by falling back to `"assembler"` when `args.Exports` isn't set.

> Should we could also refuse jobs with `len(args.Exports)>1`? If someone is requesting two artifacts and we return only one, something is not right...

There's no mechanism currently to request multiple exports (I think?).  I suppose this could happen if a newer (future) composer is paired with an older (this version) worker.  If that's a possibility, then we would have to fail the request, yes.

Reply:
That's something we probably want to be prepared for.

Reply:
Covering both cases now.